.copywrite.hcl

```diff
@@ -0,0 +1,14 @@
+schema_version = 1
+
+project {
+  license        = "MPL-2.0"
+  copyright_year = 2018
```

Comment:
Not a big deal, but do you think we should bump this up? I wonder where 2018 came from 🤔

Reply:
The copywrite script suggested it. I assumed it was when the repo was created.